### PR TITLE
Remove specialized comment AJAX

### PR DESCRIPTION
**Pull request by crhallberg**
With the new Lightbox path, we are able to remove the custom AJAX method for adding comments to records.
- [ ] Look into suppressing lightbox success message.

### CODE REVIEW

File: themes/bootstrap3/js/record.js [hunk 1]
```diff
@@ -78,64 +84,10 @@ function refreshCommentList($target, recordId, recordSource) {
...
-    return false;
   });
-  // Delete links
-  $('.delete').click(function commentDeleteClick() {
-    var commentId = this.id.substr('recordComment'.length);
-    deleteRecordComment(this, $('.hiddenId').val(), $('.hiddenSource').val(), commentId);
-    return false;
-  });
-  // Prevent form submit
-  return false;
```
Comment (crhallberg) [inline themes/bootstrap3/js/record.js]: This should not have been deleted, will restore.

File: themes/bootstrap3/templates/RecordTab/usercomments.phtml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 <div class="comment-list">
   <?=$this->render('record/comments-list.phtml')?>
 </div>
-<form class="comment-form" name="commentRecord" action="<?=$this->recordLink()->getActionUrl($this->driver, 'AddComment')?>" method="post">
+<form class="comment-form" name="commentRecord" action="<?=$this->recordLink()->getActionUrl($this->driver, 'AddComment')?>" method="post" data-lightbox data-lightbox-onclose="addRecordCommentCallback">
```
Comment (crhallberg) [inline themes/bootstrap3/templates/RecordTab/usercomments.phtml]: data-lightbox-onsubmit cannot be used to suppress the lightbox popup... I don't know if we have a mechanism for that.

File: themes/bootstrap3/js/record.js [hunk 3]
```diff
@@ -78,64 +84,10 @@ function refreshCommentList($target, recordId, recordSource) {
       return false;
     });
     $target.find('.comment-form input[type="submit"]').button('reset');
-    if (typeof grecaptcha !== 'undefined') {
-      grecaptcha.reset();
```
Comment (crhallberg) [inline themes/bootstrap3/js/record.js]: This was a duplicate reset.

**Comment by demiankatz** [comment]
@crhallberg, this one hasn't moved in a while. What is its status?

**Comment by crhallberg** [comment]
I think we should close it for now, it's no longer high on my list and is quickly falling behind.

**Comment by demiankatz** [comment]
Fine with me. It's functional as-is. We can revisit later.

**Pull request closed without merging**